src/Runtime.Base/src/System/Runtime/__Finalizer.cs

```diff
@@ -22,6 +22,10 @@ internal static class __Finalizer
         [NativeCallable(EntryPoint = "ProcessFinalizers", CallingConvention = CallingConvention.Cdecl)]
         public static void ProcessFinalizers()
         {
+#if INPLACE_RUNTIME
```

Comment:
Inplace-runtime used for CoreRT was not initializing COM on the finalizer thread at all. This is fixing it.